### ISSUE TITLE
[1.18.2] Can't use drawers for the "Upgrade Template"

### ISSUE THREAD

**Issue by INooBTooMuch** [eutro/FramedCompactDrawers#13]
Minecraft : 1.18.2
Framed Compact Drawers : 1.18-4.1.0
Forge : 40.1.68

Can't use Framed Drawers (at least the 2x2 that I tried) to make the "Upgrade Template". Also can't use the "Usable" drawer after doing the recipe with the "Framed" one.

**Comment by eutro** [comment]
> Can't use Framed Drawers (at least the 2x2 that I tried) to make the "Upgrade Template".

Right, the framed drawers aren't part of the `storagedrawers:drawers` tag (which I can fix).

> Also can't use the "Usable" drawer after doing the recipe with the "Framed" one.

I'm not sure what you mean.

**Comment by INooBTooMuch** [comment]
the framed drawers cant fit items but after you put the filler blocks for the texture they are usable, you cant use either of those for the recipe

**Comment by eutro** [comment]
Right, yeah, you can't use textured framed drawers either. Thank you.
